src/axom/inlet/SphinxDocWriter.cpp

```diff
@@ -121,16 +106,110 @@ void SphinxDocWriter::writeTable(const std::string& title,
 }
 
 void SphinxDocWriter::writeAllTables() {
-  for (unsigned int i = 0; i < m_rstTables.size(); i++) {
-    writeSubtitle(m_rstTables[i].tableName);
-    if (m_rstTables[i].description != "") {
-      m_oss << "Description: " << m_rstTables[i].description << std::endl << std::endl;
+  for (std::string& pathName : m_inletTablePathNames) {
+    writeSubtitle(m_rstTables[pathName].tableName);
+    if (m_rstTables[pathName].description != "") {
+      m_oss << "Description: " << m_rstTables[pathName].description 
+            << std::endl << std::endl;
     }
-    if (m_rstTables[i].rstTable.size() > 1) {
-      writeTable("Fields", m_rstTables[i].rstTable);
+    if (m_rstTables[pathName].rstTable.size() > 1) {
+      writeTable("Fields", m_rstTables[pathName].rstTable);
     }
   }
+}
+
+std::string SphinxDocWriter::getDefaultValueAsString(axom::sidre::View* view) {
+  axom::sidre::TypeID type = view->getTypeID(); 
+  if (type == axom::sidre::TypeID::INT8_ID) {
+    int8 val = view->getData();
+    return val ? "True" : "False";
+  } else if (type == axom::sidre::TypeID::INT_ID) {
+    int val = view->getData();
+    return std::to_string(val);
+  } else if (type == axom::sidre::TypeID::DOUBLE_ID) {
+    double val = view->getData();
+    return std::to_string(val);
+  } 
+  return view->getString();
+}
+
+std::string SphinxDocWriter::getRangeAsString(axom::sidre::View* view) {
+  std::ostringstream oss;
+  oss.precision(3);
+  oss << std::scientific;
+
+  axom::sidre::TypeID type = view->getTypeID();
+  if (type == axom::sidre::INT_ID) {
+    int* range = view->getArray();
+    oss << range[0] << " to " << range[1];
+  } else {
+    double* range = view->getArray();
+    oss << range[0] << " to " << range[1];
+  }
+  return oss.str();
+}
+
+std::string SphinxDocWriter::getValidValuesAsString(axom::sidre::View* view) {
+  // auto
+  SLIC_WARNING_IF(view->getTypeID() != axom::sidre::INT_ID,
+                  "discrete range is only valid for integers");

```

Comment:
I am not sure what "// auto" is so either clarify it or remove it (my guess is its a left over from copy/paste)

Shouldn't we be supporting valid values of doubles?  Is there a precision problem you are worried about? Also the placement of this warning makes this warning a bit odd sounding if the users ever hits it.  Hopefully they wouldn't though.

"discrete range"  to "valid values"